gratipay/models/participant/packages.py

```diff
@@ -6,6 +6,61 @@
 
 class Packages(object):
 
+    def get_packages_for_claiming(self, manager):
+
+        """Return a list of packages on the named ``manager`` for which the
+        participant has verified an email address on Gratipay, along with the
+        current package owner on Gratipay (if any).
+
+        :param string manager: the name of the package manager on which to look
+            for potential packages
+
+        :return: a list of (:py:class:`~gratipay.models.package.Package`,
```

Comment:
This doc looks outdated - looks like we're returning tuples with 4 elements, and not 3. The `is_primary` element seems to be missing.